### PR TITLE
Bug 1793750: Fix ds force rollout

### PR DESCRIPTION
**Pull request by gnufied**
Ensure that daemonsets are rolled out when something in LocalVolume object changes.

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1793750

### CODE REVIEW

File: pkg/controller/controller_test.go [hunk 1]
```diff
@@ -73,7 +78,8 @@ func (s *fakeApiUpdater) applyStorageClass(sc *storagev1.StorageClass) (*storage
 }
 
 func (s *fakeApiUpdater) applyDaemonSet(ds *appsv1.DaemonSet, expectedGeneration int64, forceRollout bool) (*appsv1.DaemonSet, bool, error) {
-	s.daemonSets = append(s.daemonSets, ds)
+	dsRollout := daemonSetRollout{forceRollout: forceRollout, daemonSet: ds}
```
Comment (jsafrane) [inline pkg/controller/controller_test.go]: I can see `forceRollout` only written, but it does not seem to be read anywhere in the test

Comment (gnufied) [reply pkg/controller/controller_test.go]: fixed

**Pull request merged**